lektor/quickstart.py

```diff
@@ -266,3 +269,78 @@ def plugin_quickstart(defaults=None, project=None):
         'author_name': author_name,
         'author_email': author_email,
     }, path)
+
+
+def theme_quickstart(defaults=None, project=None):
+    if defaults is None:
+        defaults = {}
+
+    g = Generator('theme')
+
+    theme_name = defaults.get('theme_name')
+    if theme_name is None:
+        theme_name = g.prompt('Theme Name', default=None,
+            info='This is the human readable name for this theme')
+
+    theme_id = theme_name.lower()  # pylint: disable=no-member
+    if theme_id != 'lektor' and theme_id.startswith('lektor'):
+        theme_id = theme_id[6:].strip()
+    if theme_id != 'theme' and theme_id.startswith('theme'):
+        theme_id = theme_id[5:]
+    if theme_id != "theme" and theme_id.endswith('theme'):
+        theme_id = theme_id[:-5]
```

Comment:
This is a little more complicated than what exists for the new-plugin command. Namely it doesn't handle well the case where someone just uncreatively names their plugin `lektor plugin`. I bet you can guess how I found that out. That should be updated similarly, but I'll do that in a separate PR.